modular_skyrat/modules/container_emotes/code/container_emotes.dm

```diff
@@ -0,0 +1,74 @@
+/mob/living/verb/container_emote()
+	set name = "Emote Using Vehicle/Container"
+	set category = "IC"
+
+	if (isturf(src.loc))
+		to_chat(src, span_danger("You are not within anything!"))
+		return
+	if (loc && (!src.IsUnconscious()))
+		usr.emote("exme")
+
+/datum/emote/container_emote
+	key = "exme"
+	key_third_person = "exme"
+	message = null
+
+/datum/emote/container_emote/run_emote(mob/living/user, params, type_override = null, intentional = TRUE)
+	var/container_message
+	var/container_emote = params
+	if(QDELETED(user))
+		return FALSE
+	if(is_banned_from(user, "emote"))
+		to_chat(user, "You cannot send emotes (banned).")
+		return FALSE
+	else if(user.client?.prefs?.muted & MUTE_IC)
+		to_chat(user, "You cannot send IC messages (muted).")
+		return FALSE
+	else if (isturf(user.loc))
+		to_chat(user, "You are not within anything!")
+		return FALSE
+	else if(!params)
+		container_emote = stripped_multiline_input(user, "Choose an emote to display.", "Container Emote" , null, MAX_MESSAGE_LEN)
+		if(!container_emote)
+			return FALSE
+		var/type = input("Is this a visible or hearable emote?") as null|anything in list("Visible", "Hearable")
```

Comment:
why hearable and not just audible

Reply:
Because uhhhhhhhhh

(this is copy and pasted)